tests/Http/TagsTest.php

```diff
@@ -160,4 +162,310 @@ public function testWithoutTagScope()
 
         $this->assertEquals(19, $postsQuery->count());
     }
+
+    /**
+     * Test that a POST request to /tags with a good submission adds the staff fave badges.
+     *
+     * POST /v3/posts/:post_id/tag
+     * @return void
+     */
+    public function testGoodSubmissionEarnsABadge()
+    {
+        $post = factory(Post::class)->create();
+        $userId = $post->northstar_id;
+        $user = User::findOrFail($userId);
```

Comment:
You could load these directly from the Eloquent relationship to save a little work:

```suggestion
        $user = $post->user;
```